qcelemental/molparse/to_string.py

```diff
@@ -152,13 +152,29 @@ def to_string(molrec: Dict,
 
         atoms = _atoms_formatter(molrec, geom, atom_format, ghost_format, width, prec, 2)
 
-        first_line = f"""{{{umap.get(units.lower())}}}"""
-        second_line = """geometry={"""
+        smol = []
+
+        # Don't orient the molecule if asked to fix_com or fix_orientation
+        if molrec['fix_orientation'] or molrec['fix_com']:
+            smol.append('{orient,noorient}')
+
+        # Have no symmetry if asked to fix_symmetry
+        if 'fix_symmetry' in molrec.keys() and molrec['fix_symmetry'] == 'c1':
+            smol.append('{symmetry,nosym}')
+        else:
```

Comment:
I think this is:
```
if 'fix_symmetry' not in molrec:
    smol.append('{symmetry,nosym}')
```

so the fix_symmetry, non-c1 case gets nothing. does this sound sensible to you?

Reply:
Yes that sounds reasonable to me.